iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/JobClientTests.java

```diff
@@ -81,6 +113,10 @@ public static void setUp() throws IOException, IotHubException, InterruptedExcep
         isBasicTierHub = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue(TestConstants.IS_BASIC_TIER_HUB_ENV_VAR_NAME));
         isPullRequest = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue(TestConstants.IS_PULL_REQUEST));
 
+        tenantId = Tools.retrieveEnvironmentVariableValue(TestConstants.IOTHUB_TENANT_ID_ENV_VAR_NAME);
+        clientId = Tools.retrieveEnvironmentVariableValue(TestConstants.IOTHUB_CLIENT_ID_ENV_VAR_NAME);
```

Comment:
Why declare these static fields outside and initialize them in the setUp?

Reply:
I mean why not declare and initialize at the same time?

Reply:
Just following the pattern of the other environment variables. Would you rather they be retrieved inline with their declarations?

Reply:
I see, I don't really have a strong opinion, but I think it's standard for static fields to be initialized inline.

Reply:
I moved these to local variables when I factored out the TokenCredential construction, so this discussion is now moot